scripts/pre_commit_linter.py

```diff
@@ -719,7 +727,9 @@ def _check_comments(all_files):
                     # Check that the comment ends with the proper punctuation.
                     if (line[-1] not in
                             ALLOWED_TERMINATING_PUNCTUATIONS) and (
-                                not any(word in line for word in EXCLUDED_PHRASES)):
+                                not any(
```

Comment:
Here and below -- hard to read. Pull intermediate clauses out into variables and refer to those in the if clause instead.

Reply:
Done.